REFERENCE.md

```diff
@@ -274,6 +274,24 @@ For further discussion of the overlap assumption, please see Imbens and Rubin (2
 
 While the algorithm in `regression_forest` is very similar to that of classic random forests, it has several notable differences, including 'honesty', group tree training for variance estimates, and restrictions during splitting to avoid imbalanced child nodes. These features can cause the predictions of the algorithm to be different, and also lead to a slower training procedure than other packages. We welcome GitHub issues that shows cases where GRF does notably worse than other packages (either in statistical or computational performance), as this will help us choose better defaults for the algorithm, or potentially point to a bug.
 
+
+### Forests predict different values depending on the platform even though the seed is the same
+
+When it comes to cross-platform predictions, the output of `grf` forest will depend on a few factors beyond the forest seed.
+
+One such factor is the compiler that was used to build `grf`. Different compilers may have optimization procedures that perform more or less aggressive floating-point optimizations, and these could lead to slightly different forest splits if the data requires numerical precision. If your data contains variables have more than 8 significant digits, please consider rounding these variables, or multiplying them by a constant.
```

Comment:
I don't think I would describe these as 'optimization procedures' that perform 'aggressive floating-point optimizations', but rather 'different default behaviors around floating point rounding'.

Also I'm not sure if we should recommend multiplying variables by a constant. I think you would only want to do this for the `X` values (and not `Y`, `W`, etc.), and in this case you could still end up with slightly different estimates because of the differences in the observation variables?

Reply:
Thanks, will edit accordingly.